peachjam/templates/peachjam/_document_table.html

```diff
@@ -4,8 +4,8 @@
     <thead>
       <tr>
         <th scope="col" style="width: 70%">{% trans 'Title' %}</th>
-        {% if doc_table_show_jurisdiction %}
```

Comment:
What was the logic to determine the value of this variable previously?

Reply:
It's set only in **africanlii** to show the jurisdiction of taxonomies alone. But if we're showing jurisdiction across the site, I didn't think it'd matter.

Reply:
Agreed. Can you remove the logic that sets it in views.py or wherever?

Reply:
![Screenshot 2024-01-22 at 11 53 32](https://github.com/laws-africa/peachjam/assets/52611827/f573dcc0-f0f8-4876-a938-b33a451410de)